crates/biome_js_analyze/src/lib.rs

```diff
@@ -258,7 +258,7 @@ mod tests {
             String::from_utf8(buffer).unwrap()
         }
 
-        const SOURCE: &str = r#"<a href="class/html-css1/navigation/links#" onclick="window.location.href=index.html"> Home </a>
+        const SOURCE: &str = r#"<>{provider}</>
```

Comment:
I think these changes don't need to be checked in.

Reply:
It's fine :) This is an internal test that people can use to quickly test their code. It's OK if changes are checked in